src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Button/ButtonHandler.cs

```diff
@@ -1,46 +1,25 @@
-﻿using System;
-using Xamarin.Forms;
-
-#if __IOS__
-using NativeView = UIKit.UIButton;
-#elif __MACOS__
-using NativeView = AppKit.NSButton;
-#elif MONOANDROID
-using NativeView = AndroidX.AppCompat.Widget.AppCompatButton;
-#elif NETSTANDARD
-using NativeView = System.Object;
-#elif WINDOWS
-using NativeView = Microsoft.UI.Xaml.Controls.Button;
-#endif
-
-namespace Xamarin.Platform.Handlers
+﻿namespace Xamarin.Platform.Handlers
 {
-	public class ButtonHandler : AbstractViewHandler<IButton, NativeView>
+	public partial class ButtonHandler
 	{
 		public static PropertyMapper<IButton, ButtonHandler> ButtonMapper = new PropertyMapper<IButton, ButtonHandler>(ViewHandler.ViewMapper)
 		{
-			[nameof(IButton.Text)] = MapText,
-			[nameof(IButton.Color)] = MapColor
-		};
-
-		public static void MapColor(ButtonHandler handler, IButton button)
-		{
-			ViewHandler.CheckParameters(handler, button);
-			handler.TypedNativeView?.UpdateColor(button);
-		}
-
-		public static void MapText(ButtonHandler handler, IButton button)
-		{
-			ViewHandler.CheckParameters(handler, button);
-			handler.TypedNativeView?.UpdateText(button);
-		}
-
 #if MONOANDROID
-		protected override NativeView CreateNativeView() => new NativeView(this.Context);
-#else
-		protected override NativeView CreateNativeView() => new NativeView();
+			[nameof(IButton.BackgroundColor)] = MapBackgroundColor,
```

Comment:
It is not really necessary now. There is specific code in the Renderer on Android and that's why I added it. Although since it is not used now, I have removed it.